app/assets/stylesheets/application.scss

```diff
@@ -24,15 +24,28 @@
 }
 
 .bg-ambiance {
-  background-image: url("https://images.pexels.com/photos/1565982/pexels-photo-1565982.jpeg");
+  background-image: url("https://github.com/mikez321/file_store/blob/master/dark_bg_food.png?raw=true");
```

Comment:
Also an alternative to what I did of putting transparent blocks behind important text